components/form/profilesForm.tsx

```diff
@@ -5,13 +5,25 @@ import * as React from 'react';
 import * as Yup from 'yup';
 
 const validationSchema = Yup.object().shape({
-    linkedin_url: Yup.string().url('Invalid url').required('Required'),
-    github: Yup.string().url('Invalid url').required('Required'),
-    leetcode_url: Yup.string().url('Invalid url'),
-    hackerrank_url: Yup.string().url(),
-    codepen_url: Yup.string().url(),
+    linkedin_url: Yup.string().matches(
+        /((https?:\/\/)?((www|\w\w)\.)?linkedin\.com\/)((([\w]{2,3})?)|([^\/]+\/(([\w|\d-&#?=])+\/?){1,}))$/,
+        'Invalid linkedIn url'
+    ),
+    github: Yup.string().matches(
+        /^(https?:\/\/)?(www\.)?github\.com\/[a-zA-Z0-9_]{1,25}$/gim,
```

Comment:
@JRakin can you please move the checks to a common helper? we can re-use them in other pages.